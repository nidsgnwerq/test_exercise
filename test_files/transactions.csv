type,client,tx,amount
deposit,1,10001,2.000
deposit,2,20001,25
deposit,3,30001,500.000
deposit,6,60001,10000.0000
deposit,7,70001,300.000
withdrawal,1,10002,1.000
dispute,2,20001
resolve,2,20001
dispute,6,60001
chargeback,6,60001
chargeback,6,60002
withdrawal,2,10002,20
withdrawal,3,10002,100.000
deposit,4,40001,10000
withdrawal,7,70001,100
