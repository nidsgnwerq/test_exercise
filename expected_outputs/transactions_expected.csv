client, available, held, total, locked
1, 1.0, 0, 1.0, False
2, 5.0, 0.0, 5.0, False
3, 400.0, 0, 400.0, False
4, 10000.0, 0, 10000.0, False
6, 0.0, 0.0, 0.0, True
7, 200.0, 0, 200.0, False
